src/Cardano/CoinSelection/LargestFirst.hs

```diff
@@ -59,7 +68,7 @@ import qualified Data.Map.Strict as Map
 --      into /descending order of coin value/.
 --
 --      The /head/ of the list is always the unpaid output with the
---      /greatest coin value/.
+--      /largest coin value/.
```

Comment:
This might seem like a trivial change, but the wording is now consistent with the name of the algorithm.